drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/BrowserCapabilitiesList.java

```diff
@@ -230,5 +222,27 @@ public int getPrecedence() {
         }
     }
 
-    ;
+    public static class chromeHeadless implements BrowserCapabilities {
+        @Override
+        public String getReadableName() {
+            return "chromeHeadless";
+        }
+
+        @Override
+        public String getImplementationClassName() {
+            return "org.openqa.selenium.chrome.ChromeDriver";
+        }
+
+        @Override
+        public Map<String, ?> getRawCapabilities() {
+            DesiredCapabilities headlessChrome = DesiredCapabilities.chrome();
+            headlessChrome.setBrowserName("chromeHeadless");
```

Comment:
Had to set the `browserName` here, otherwise, when DesiredCapabilities class is called internally, the browserName gets overridden somehow to `chrome`, and it no longer functions in the headless mode but in the normal way.